inst/include/cpp11/protect.hpp

```diff
@@ -159,8 +149,12 @@ void unwind_protect(Fun code) {
     throw unwind_exception(token);
   }
 
-  R_UnwindProtect(&internal::unwind_protect_unwrap_void<Fun>, &code, internal::maybe_jump,
-                  &unwind_data, token);
+  (void)R_UnwindProtect(
+      [](void* data) -> SEXP {
+        Fun* callback = (Fun*)data;
+        return R_NilValue;
```

Comment:
I think you are missing the call to callback here right?

```
(*callback)();
```

Reply:
yep, oops